paper.tex

```diff
@@ -106,8 +106,9 @@ \section*{Introduction}
 and show how it derives from (and is limited by) the coalescent with
 recombination.
 \item Suggest a definition of ARGs that is free of these limitations,
-where the formulation is derived from a diploid pedigree rather than
-the coalescent approximations and spatial annotations are attached to
+where the formulation is derived from a pedigree rather than from
```

Comment:
Again, I think we have to be specific here or people won't think it through properly. Putting "diploid" in there makes people think about the multiple genomes explicitly. I know there's no particular reliance of a given ploidy level, but that's something we can clarify later.

Genetic material is too vague as well, we need something that gets across the idea that we're talking about the actual genome coordinates.

Reply:
The formulation need not be derived from a standard diploid pedigree, though? It would be just as reasonable to derived it from e.g. a gametophytic system, I think. So we just want to be careful here not to tie it too strongly to the human-style pedigree, I think?

Reply:
As I say, I know there's no strong dependence on particular ploidy levels here. This isn't a mature draft and I never even remotely thought of this text as being final.

Reply:
Yep, absolutely. I know this is a draft section, I just thought it worth changing the language so that it didn't end up going into the final text by default/accident. But happy to revery and reconsider when the final text is done.